src/hardware/motor.cpp

```diff
@@ -184,14 +184,17 @@ float StepperMotor::getMicrostepAngle() const {
 // Set if the motor direction should be reversed or not
 void StepperMotor::setReversed(bool reversed) {
 
-    // Set if the motor should be reversed
-    this -> reversed = reversed;
+    if (reversed)
```

Comment:
Why use an integer instead of a bool? Using a bool simplifies the logic and only takes 8 bits

Reply:
Generally
```
x = y * reversed
```
gives shorter assembler code than
```
if (reversed)
    x = -y
```

Reply:
How much shorter though? Is it really worth it?